demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -101,13 +91,29 @@ def docker_tag_to_runfiles(
     Returns: A dict of image to List of files(Tuple[path, obj]) including native images
 
     """
+    docker_flags = set(docker_image_flag.split(","))
     tags_to_files = defaultdict(list)
     for file, obj in files_to_run:
         if not obj:
             continue
-        for docker_image in obj.docker_images:
-            tags_to_files[docker_image].append((file, obj))
-    return with_native_tags(tags_to_files, docker_image_flag)
+        else:
+            for image in obj.docker_images:
+                tags_to_files[image].append((file, obj))
+    return with_native_tags(tags_to_files, docker_flags, docker_image)
+
+
+@lru_cache()
+def get_docker_python_path() -> str:
+    """
+    precommit by default mounts the content repo to source.
+    This means CommonServerPython's path is /src/Packs/Base/...CSP.py
+    Returns: A PYTHONPATH formatted string
+    """
+    path_to_replace = str(Path(CONTENT_PATH).absolute())
```

Comment:
absolute compared to cwd?

Reply:
this is not related to this PR, moved to its original place to prevent the git diff